deluge/tests/test_common.py

```diff
@@ -53,15 +53,17 @@ def test_fpeer(self):
         self.assertTrue(fpeer(10, -1) == '10')
 
     def test_ftime(self):
-        self.assertTrue(ftime(0) == '')
-        self.assertTrue(ftime(5) == '5s')
-        self.assertTrue(ftime(100) == '1m 40s')
-        self.assertTrue(ftime(3789) == '1h 3m')
-        self.assertTrue(ftime(23011) == '6h 23m')
-        self.assertTrue(ftime(391187) == '4d 12h')
-        self.assertTrue(ftime(604800) == '1w 0d')
-        self.assertTrue(ftime(13893086) == '22w 6d')
-        self.assertTrue(ftime(59740269) == '1y 46w')
+        assert ftime(0) == ''
```

Comment:
Using python assert is for pytest and will not output traceback with trial on failure so please revert this change. The assertTrue could be replaced with assertEquals but it's not really worth unless changing whole test file...

Reply:
Isn't pytest what's being used now?

Reply:
Well, both are in use and I can't remember what the end-goal was... It doesn't actually test the twisted deferreds properly so would need this plugin: https://github.com/pytest-dev/pytest-twisted

The point is that changing it for one subset of tests is not a good idea. The tests themselves are subclassing trial unittest so best to keep that compatibility.

Reply:
No prob. I thought the intention was to switch over to pytest, and had introduced a failure that wasn't giving me any good info with assertTrue.

Reply:
Sure it would be good to move to pytest entirely but it's better to do it as part of an overall change rather than here and there and potentially cause more headaches.